test/Storyshoots.spec.js

```diff
@@ -1,6 +1,47 @@
 import initStoryshots from '@storybook/addon-storyshots'
 import './env'
 
+jest.mock('../src/cxs', () => i => {
```

Comment:
is it worth testing this in isolation beyond actually generating snapshots?

Reply:
There's a case to be made there for sure, honestly this feels more like a quick hack to get free css snapshot. We could also just return `"className"` and have separate css snapshots.

Since (afaik) there's no readily available solution for CXS snapshots and we would need to implement our own for something we already have here it has a very low ROI imo.